tests/transactional/disable_selinux.pm

```diff
@@ -0,0 +1,59 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Package: transactional-update
+# Summary: Enable SELinux on transactional server
+#
+# Maintainer: QA-C team <qa-c@suse.de>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use transactional qw(process_reboot);
+use version_utils qw(is_leap_micro is_microos is_sle_micro is_public_cloud);
+use Utils::Systemd qw(systemctl);
+use Utils::Logging 'save_and_upload_log';
+use transactional qw(trup_call check_reboot_changes process_reboot);
+use publiccloud::utils qw(allow_openqa_port_selinux);
+use bootloader_setup 'replace_grub_cmdline_settings';
+
+
+sub check_disabled {
+    record_info('SELinux', script_output('sestatus'));
+    assert_script_run('! selinuxenabled');
+    validate_script_output("getenforce", sub { m/Disabled/ });
+    validate_script_output("sestatus", sub { m/SELinux status:.*disabled/ });
+}
```

Comment:
This function is an exact duplicate of https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/transactional/enable_selinux.pm#L33-L38

Perhaps we should move this function to some library, but I honestly don't know which.

Reply:
Can you check the cnages? Thank you.

Reply:
CI is failing...

Reply:
done